frontend/src/utils/CampUtils.ts

```diff
@@ -483,3 +483,26 @@ export const getFormattedCampDateRange = (camp: CampResponse): string => {
     ? `${format(firstCampDate, "PP")} - ${format(lastCampDate, "PP")}`
     : "";
 };
+
+export const checkDatesInRange = (
```

Comment:
I think we can simplify this function a bit - smthn like this: 

```
export const checkDatesInRange = (
  rangeDates: string[],
  datesToCheck?: Date[],
): boolean => {
  if (!datesToCheck || datesToCheck.length === 0) return false;

  const minDate = Math.min(...rangeDates.map((date) => Date.parse(date)));
  const maxDate = Math.max(...rangeDates.map((date) => Date.parse(date)));

  return datesToCheck.some(
    (date: Date) =>
      new Date(date).getTime() >= minDate &&
      new Date(date).getTime() <= maxDate,
  );
};
```

Reply:
Works like a charm, thanks!